File: front/src/app/models/Estadistica.ts
export class Estadistica {
	profesores: number;
	alumnos: number;
	cursos: number;
	clases: number;

	constructor(_profesores: number, _alumnos: number, _cursos: number, _clases: number) {
		this.profesores = _profesores;
		this.alumnos = _alumnos;
		this.cursos = _cursos;
		this.clases = _clases;
	}
}

File: sov_school_back/back_base/src/main/java/com/sovereingschool/back_base/DTOs/Estadistica.java
package com.sovereingschool.back_base.DTOs;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class Estadistica {
    int profesores;

    Long alumnos;

    Long cursos;

    Long clases;
}
